src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.FileStream.cs

```diff
@@ -2,152 +2,299 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
 using MicroBenchmarks;
 
 namespace System.IO.Tests
 {
     [BenchmarkCategory(Categories.Libraries)]
     public class Perf_FileStream
     {
-        private const int DefaultBufferSize = 4096;
-        
-        [Params(DefaultBufferSize / 8, 200000)]
-        public int BufferSize;
+        private const int OneKiloByte = 1_000;
```

Comment:
May be interesting to review "real world" values:

https://grep.app/search?q=new%20FileStream.%2A0x&regexp=true&filter[lang][0]=C%23
https://grep.app/search?q=new%20FileStream.%2A%5Cd%7B4%2C%7D&regexp=true&filter[lang][0]=C%23
https://grep.app/search?q=new%20FileStream.%2Asize%3A%5Cs%3F.%5B%5E4%5D%5Cd%7B3%2C%7D&regexp=true&filter[lang][0]=C%23

0x1000 / 4096 dominate (redundantly)
I also see 0x1_000 and )x16, 2000, 1024, 512 << 10, 64 * 1024